### PR TITLE
Update SHOW SETTINGS table to include `dbms.cluster.network.connect_timeout`

### PR DESCRIPTION
**Pull request by JPryce-Aklundh**
Failure resolved when https://github.com/neo-technology/neo4j/pull/23939  is merged

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-862.surge.sh

Updated pages:
- [SHOW SETTINGS](https://neo4j-docs-cypher-862.surge.sh/cypher-manual/5/clauses/listing-settings)

File: modules/ROOT/pages/clauses/listing-settings.adoc [hunk 1]
```diff
@@ -227,6 +227,10 @@ LIMIT 10
 | "20s"
 | "Time out for protocol negotiation handshake."
 
+| "dbms.cluster.network.connect_timeout"
+| "30s"
+| "The maximum amount of time we wait to establish a network connection."
```
Comment (renetapopova) [inline modules/ROOT/pages/clauses/listing-settings.adoc]: I've updated the description of this setting. See https://trello.com/c/aWrYCE5Y/213-document-dbmsclusternetworkconnecttimeout

Comment (renetapopova) [reply modules/ROOT/pages/clauses/listing-settings.adoc]: And also, I think it wil be introduced in 5.17.

Comment (JPryce-Aklundh) [reply modules/ROOT/pages/clauses/listing-settings.adoc]: Yes, it is a 5.17 feature. But this table only shows what is returned by the show settings command (so no release info etc.)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-862.surge.sh

Updated pages:
- [SHOW SETTINGS](https://neo4j-docs-cypher-862.surge.sh/cypher-manual/5/clauses/listing-settings)

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

**Pull request merged**